raw_radar_reader/README.md

```diff
@@ -0,0 +1,72 @@
+# Raw Radar Interface
+
+This directory contains tools for interacting with raw radar data. Zendar
+makes available raw ADC samples from our FMCW radars. The example tools
+in this directory allow researchers to easily use this data format.
+
+o## Dependencies
+```
+libprotobuf
+protobuf-compiler
+```
+
+Python packages:
+```
+numpy
+matplotlib
+gzip
+os
+struct
+protobuf
+argparse
+```
+
+## Setup
+
+To use these tools, perform the following steps:
+```
+cd ./protocol
+protoc primitive.proto --python_out .
+protoc radar.proto --python_out .
+protoc acquisition.proto --python_out .
+cd ../
+export PYTHONPATH=${PYTHONPATH}:<path-to-ZendarSDK>/raw_radar_reader/protocol
```

Comment:
can this not be added into the environment.sh script such that we have all documentation at one place?

Reply:
Good idea, I added a environment.sh specific to this project (since it's standalone).